test/conformance/gateway_conformance_test.go

```diff
@@ -25,8 +24,9 @@ import (
 )
 
 var (
-	showDebug     = true
-	shouldCleanup = true
+	showDebug                  = true
+	shouldCleanup              = true
+	enableAllSupportedFeatures = true
```

Comment:
Nit: could we make these variables consts?

Reply:
👍